internal/agent/device/lifecycle/manager.go

```diff
@@ -97,6 +98,14 @@ func (m *LifecycleManager) Initialize(ctx context.Context, status *v1alpha1.Devi
 		}
 	}
 
+	_, updateErr := m.statusManager.Update(ctx, fcstatus.SetDeviceLifecycleStatus(v1alpha1.DeviceLifecycleStatus{
+		Status: "Enrolled",
+		Info:   lo.ToPtr("Device was enrolled"),
```

Comment:
nit: please use util.FromPtr

Reply:
Looks like that was removed in [this commit](https://github.com/flightctl/flightctl/commit/5198e4cad6e32fd7d0d118fe6013f9721df732a3)

Reply:
sigh